public/themes/default/css/editor.css

```diff
@@ -1 +1,18 @@
 @import "./editor/editor.css";
+
+.ribbon {
```

Comment:
This is an admin-only bit of CSS when running with `--admin-mode` but it should probably be in its own file, and then templated in only when that flag is active.